plenum/common/messages/fields.py

```diff
@@ -90,21 +90,18 @@ def _specific_validation(self, val):
             return 'empty string'
 
 
-class LimitedLengthStringField(NonEmptyStringField):
+class LimitedLengthStringField(FieldBase):
     _base_types = (str,)
 
     def __init__(self, max_length: int, **kwargs):
-        if max_length < 1:
-            raise Exception('Incorrect validation parameters')
-
```

Comment:
Let's leave here an assertion:
`assert max_length > 0, "should be greater than 0"`
Looks like it makes sense.